docs/kb/garbage-collection-ics.rst

```diff
@@ -0,0 +1,61 @@
+==============================================
+Efficient Tombstone Garbage Collection in ICS
```

Comment:
nit: please fix typo s/timbstone/tombstone/ in patch and PR titles

Reply:
ahh, sorry! i missed this comment. they were inherited from the original commit introducing this file, so i kept them.